### DISCUSSION TITLE
Question about creating vocab of Problem 2.

### DISCUSSION THREAD

**Discussion by HayeonLee** [seominjoon/kaist-ai605#116] [Q&A] [answered]
When we create vocab, could we use both texts of 'context' and 'question'? 
 When I created vocab by only using 'context' texts, key error occurred since a few words of questions are not included in the vocab. 
 Using 'UNK' may be one of the choices, yet, you said that [using UNK for tf-idf will have bad side effects](https://github.com/seominjoon/kaist-ai605/discussions/98#discussioncomment-1518259). 
 Is it okay to use texts of 'question's to make vocab?
 Thanks in advance.

**Comment by seominjoon** [comment, marked as answer]
In TF-IDF, after you create vocab, you should just ignore words that are not in the vocab, instead of mapping them to UNK.
